src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,63 @@
+.calendar {
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
+  padding: 0 10px;
+  display: flex;
+
+  &__day {
+    align-items: center;
+    box-sizing: border-box;
+    color: black;
+    background-color: $day-color;
+    width: $day-size;
+    height: $day-size;
+    border: $border-size solid black;
+    display: flex;
+    justify-content: center;
+    position: relative;
+    transform: translateY(0);
+    transition:
+      background-color $animation-duration,
+      transform $animation-duration;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $hover-color;
```

Comment:
Ensure that the variable `$hover-color` is defined in your SCSS files. Without a definition, this will cause a compilation error.